packages/vite/src/types.ts

```diff
@@ -57,4 +57,11 @@ export interface VitePluginConfig<Theme extends object = object> extends UserCon
    * @default 'cors'
    */
   fetchMode?: 'cors' | 'navigate' | 'no-cors' | 'same-origin'
+
+  /**
+   * Disable `import 'uno.css'` existing check.
+   *
+   * @default false
+   */
+  disableEntryCheck?: boolean
```

Comment:
```suggestion
  checkImport?: boolean
```

Maybe use `checkImport` and default to `true`? that give us more flexibility for the feature that we can add different check modes for example.

Reply:
Consider future usage, `checkImport?: boolean | { entry?: boolean }` would be better. 
But for now, since there isn't any other import check. It's may be over designed.

Reply:
I think just rename it will be fine for now.